### DISCUSSION TITLE
Use and/or Sync with new F1 Live Timing web version

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#181] [Feature requests] [answered]
I noticed the formula1.com web site has a new Live Timing app described here:
https://www.formula1.com/en/latest/article.improved-live-timing-on-f1-com-for-the-2022-season.3Y1ipx839mv7xeteTqQ8Mk.html

Requires Access or Pro subscription. This is basically a web version of the very popular Android / iOS 'Formula 1' app. What appears to be lacking on the web is ability to use for replays, so my hope is this would be added by them in the future.

Something I was already thinking about was the English audio feature which is pretty much to a radio broadcast 'hyper descriptive format'. I think there would be benefit from being able to use this new Live Timing component, AND in some cases the audio too.  I have a feeling it's the same as the what would be used on the BBC radio broadcast.

New: https://www.formula1.com/en/f1-live-timing.html
(Old: https://www.formula1.com/en/f1-live-legacy.html)

So the purpose of the post is for ideas:

1) Add link in F1MV app (like other shortcuts on the three dot menu) 'Official F1 Live Timing' **SIMPLE**
2) Add pop-out window to Official F1 Live Timinghttps://www.formula1.com/en/f1-live-timing.html). Maybe add this on the events page, e.g. 2022 Dutch Grand Pris (Live) underneath F1MV Live Timing. This has benefit of a cleaner window, without typical browser shortcuts, etc. **SIMPLE**
3) Same as bullet 2 plus add ability to sync. **COMPLEX**
4) Similar as bullet 2 but just for the audio. This has benefit of cleaner window, without typical browser shortcuts, etc and also for users that don't really want to use the official F1 Live Timing.  Maybe add this to the F1MV Live Stats window, similar to dutch radio **COMPLEX**
5) Same as bullet 4 plus ability to sync audio **COMPLEX**

**Comment by jeppeolesen** [comment, marked as answer]
Closing this as MultiViewer now offers all of the same features as the web based official F1 live timing (and more).
